core/tracking.go

```diff
@@ -20,61 +20,61 @@ import (
 )
 
 // WithTracking wraps component input functions to support tracking of core components.
-func WithTracking(tracker Tracker) WireOption {
+func WithTracking(parentCtx context.Context, tracker Tracker) WireOption {
 	return func(w *wireFuncs) {
 		clone := *w
 
 		w.FetcherFetch = func(ctx context.Context, duty Duty, set DutyDefinitionSet) error {
 			err := clone.FetcherFetch(ctx, duty, set)
-			tracker.FetcherFetched(ctx, duty, set, err)
+			tracker.FetcherFetched(parentCtx, duty, set, err)
```

Comment:
rather remove the context from the tracker interface methods. Since it is a "post-fact" thing, ctx isn't required.

Reply:
Done!